scripts/lib/aws/apigatewayv2.sh

```diff
@@ -371,6 +379,13 @@ exports.handler = async(event) => {
 EOF
   zip authorizer.zip index.js >/dev/null
 
+  daws lambda get-function --function-name "$__function_name" >/dev/null 2>&1
+  local __status=$?
+  if [[ $__status -ne 255 && $__status -ne 254 ]]; then
```

Comment:
FYI, you don't need to check for 255 when using `daws` because `daws` standardizes our `aws` client to v2 :)